testing/zerotier-one/APKBUILD

```diff
@@ -1,38 +1,48 @@
-# Contributor: Kyle Parisi <kyleparisi@gmail.com>
-# Maintainer: Kyle Parisi <kyleparisi@gmail.com>
+# Contributor: ZeroTier, Inc. <contact@zerotier.com>
+# Maintainer: Adam Ierymenko <adam.ierymenko@zerotier.com>
+
+## Thanks to Kyle Parisi <kyleparisi@gmail.com> for doing the original package and getting it into testing
+
 pkgname=zerotier-one
-pkgver=1.1.14
-pkgrel=1
-pkgdesc="ZeroTier One allows systems to join and participate in ZeroTier virtual networks."
+pkgver=1.2.4
+pkgrel=0
+pkgdesc="Join ZeroTier virtual neteworks from your Alpine Linux system"
 url="https://www.zerotier.com/"
 arch="all"
 license="GPL3"
-depends=""
-makedepends="nodejs linux-headers"
-install=""
-subpackages="$pkgname-doc"
+depends="libstdc++"
+makedepends="linux-headers"
+install="$pkgname.pre-install $pkgname.post-deinstall"
+subpackages=""
```

Comment:
As there are manpages, please keep the `$pkgname-doc` subpackage. They will be automatically moved to it by abuild.